templates/year/data/signaturesRequest.soap

```diff
@@ -8,8 +8,7 @@
 	</SOAP-ENV:Header>
 	<SOAP-ENV:Body>
 		<GetExportRequest>
-			<qid>tex.emails</qid>
-			<parameter name="contest">SEM</parameter>
+			<contestId>SEM</contestId>
 		</GetExportRequest>
```

Comment:
Tady máš mít `<GetSignaturesRequest>` tag a `SEM` obsahuje slovně fykos/vyfuk a ty potřebuješ contest_id jakožto číslo.

Reply:
Jasně, beru zpět. Ale je to matoucí, nechceš to spíš nahradit za něco jiného než `SEM`? Protože v jednom souboru to je tak a ve druhém jinak...

Reply:
Jo, můžeme to pro přehlednost přejmenovat na něco jako `SEMID`.